### PR TITLE
feature: ZENKO-1466 convert md search doc to rst

### PR DESCRIPTION
**Pull request by rahulreddy**
### Motivation and context

Convert markdown doc to rst to publish in readthedocs

### CODE REVIEW

**Review by JianqinWang — APPROVED** [review]

File: docs/MD_SEARCH.rst [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+      </Contents>
+  </ListBucketResult>
+
+Performing Metadata Searches with Zenko
+---------------------------------------
+
+You can perform metadata searches by:
+
++ Using the :code:`search_bucket` tool in the
+  `Scality/S3 <https://github.com/scality/S3>`_ GitHub repository.
```
Comment (philipyoo) [inline docs/MD_SEARCH.rst:106]: minor: repo name change, but link still leads to same

File: docs/MD_SEARCH.rst [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
++ Creating a signed HTTP request to Zenko in your preferred programming
+  language.
+
+Using the S3 Tool
++++++++++++++++++
+
+After cloning the `Scality/S3 <https://github.com/scality/S3>`_ GitHub repository
```
Comment (philipyoo) [inline docs/MD_SEARCH.rst:113]: same here

**Review by alexanderchan-scality — APPROVED** [review]

**Comment by rahulreddy** [comment]
@bert-e approve

File: docs/MD_SEARCH.rst [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+Search requests can be also performed by making HTTP requests authenticated
+with one of the AWS Signature schemes: version 2 or version 4. \
+For more about authentication scheme, see:
+
+* https://docs.aws.amazon.com/general/latest/gr/signature-version-2.html
+* http://docs.aws.amazon.com/general/latest/gr/sigv4_signing.html
+* http://docs.aws.amazon.com/general/latest/gr/sigv4-signed-request-examples.html
+
+You can also view examples for making requests with Auth V4 in various
+languages `here <../../../examples>`__.
```
Comment (philipyoo) [inline docs/MD_SEARCH.rst:165]: link leads to 404

Comment (rahulreddy) [reply docs/MD_SEARCH.rst:165]: We have another follow up PR taking care of the leftovers.

**Pull request merged**